C4SGSvc/src/main/java/org/c4sg/constant/UserProjectStatus.java

```diff
@@ -0,0 +1,15 @@
+package org.c4sg.constant;
+
+public enum UserProjectStatus {
+
+    APPLIED("A");
```

Comment:
We need a string 'A' to be persisted on the table, So to improve readability on the code I created an Enum.